pkg/storaged/dialog.jsx

```diff
@@ -724,100 +725,50 @@ export const SelectSpaces = (tag, title, options) => {
         options,
         initial_value: [],
         hasNoPaddingTop: options.spaces.length == 0,
+        large_modal: true,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18967-20230629-132526-4d4c19dd-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)